src/test/accountExporter.spec.js

```diff
@@ -113,5 +114,54 @@ describe("accountExporter", function() {
         }
       });
     });
+
+    it("should encapsulate displayNames with commas for csv formats", function() {
+      var trailingCommas = [];
+      // The remaining empty columns, index 6 to 25.
+      for (var i = 0; i < 20; i++) {
+        trailingCommas.push(",");
+      }
+      nock("https://www.googleapis.com")
+        .post("/identitytoolkit/v3/relyingparty/downloadAccount", {
+          maxResults: 7,
+          targetProjectId: "test-project-id",
+        })
+        .reply(200, {
+          users: userList,
+          nextPageToken: "7",
+        })
+        .post("/identitytoolkit/v3/relyingparty/downloadAccount", {
+          maxResults: 7,
+          nextPageToken: "7",
+          targetProjectId: "test-project-id",
+        })
+        .reply(200, {
+          users: [],
+          nextPageToken: "7",
+        });
+
+      var result = serialExportUsers("test-project-id", {
+        format: "csv",
+        batchSize: 7,
+        writeStream: writeStream,
+      });
+      return result.then(function() {
+        expect(spyWrite.callCount).to.eq(7);
+        for (var j = 0; j < 7; j++) {
+          var expectedDisplayName = userList[j].displayName;
+          if (i === 0) {
+            expectedDisplayName = '"' + expectedDisplayName + '"';
+          }
+          var expectedEntry =
+            userList[j].localId +
+            "," +
+            userList[j].email +
+            ",false,,," +
+            expectedDisplayName +
+            trailingCommas.join("");
+          expect(spyWrite.getCall(j).args[0]).to.eq(expectedEntry + "," + os.EOL);
+        }
```

Comment:
So if the `userList` above is adjusted to be a single user with a comma in their name, do we need this loop?

Further, do we need to check this string exactly? it seems like that is done (in a complex fashion) above in the other test.

This test could probably be asserted by saying the following:

```javascript
const writeArg = spyWrite.getCall(j).args[0];
expect(writeArg).to.contain(`,"Display Name, with Comma",`);
```

Re-testing that the entire line is correct again is _probably_ overkill in this situation - then, you're testing that a multitude of things are working, instead of just one thing.

Reply:
Added one test for csv behavior and updated this test to just test comma encapsulation on single user.